common/djangoapps/student/models.py

```diff
@@ -877,6 +878,37 @@ def num_enrolled_in(self, course_id):
 
         return enrollment_number
 
+    def num_enrolled_in_exclude_admins(self, course_id):
+        """
+        Returns the count of active enrollments in a course excluding instructors, staff and CCX coaches.
+
+        Arguments:
+            course_id (CourseLocator): course_id to return enrollments (count).
+
+        Returns:
+            int: Count of enrollments excluding staff, instructors and CCX coaches.
+
+        """
+        # To avoid circular imports.
+        from roles import CourseCcxCoachRole, CourseInstructorRole, CourseStaffRole
+        course_locator = course_id
+
+        if getattr(course_id, 'ccx', None):
+            course_locator = course_id.to_course_locator()
+
+        staff = CourseStaffRole(course_locator).users_with_role()
+        admins = CourseInstructorRole(course_locator).users_with_role()
+
+        enrollment_number = super(CourseEnrollmentManager, self).get_queryset().filter(
+            course_id=course_id,
+            is_active=1,
+        ).exclude(user__in=staff).exclude(user__in=admins)
+
+        coaches = CourseCcxCoachRole(course_locator).users_with_role()
+        enrollment_number = enrollment_number.exclude(user__in=coaches)
```

Comment:
Can you add the `.exclude()` part to the end of the queryset above, at line https://github.com/mitocw/edx-platform/pull/203/files#diff-e2ffffdf6d67c9e0dee5bc10a67eb8bfR905